docs/learn/learn-account-advanced.md

```diff
@@ -330,6 +330,33 @@ derivation paths.
 
 {{ kusama: Note that you cannot import Kusama Ledger accounts in Ledger Live. To see Kusama account balances, you must import your ledger account into a [**wallet**](./wallets). :kusama }}
 
+### Account Derivation on [Subkey](#creating-accounts-with-subkey) and [Polkadot Vault](../general/polkadot-vault.md)
+
+The Subkey tool and Polkadot Vault App use the following Polkadot Standard Hard Derivation scheme:
+
+- `//network` as the primary account for `network`, named according to `network`'s named chain
```

Comment:
We should add some examples here, e.g. "For example, //polkadot is used for the Polkadot Relay Chain"